### ISSUE TITLE
possible upgrade bug

### ISSUE THREAD

**Issue by cmeinco** [elnook/notedo#12]
**Describe the bug**
error on upgrade

**To Reproduce**
notedo@notedoserver:/opt/notedo$ ./notedo upgrade
```
2020/11/25 14:59:27 New version available `0.1.8-7-gdba1ef9` (current `0.1.8`)
2020/11/25 14:59:27 Downloading: https://github.com/elnook/notedo/releases/download/0.1.8-7-gdba1ef9/notedo-linux-0.1.8-7-gdba1ef9.tar.gz
2020/11/25 14:59:32 Created backup of current binary at: /opt/notedo/.notedo/backup/bin/notedo-0.1.8.tar.gz
2020/11/25 14:59:32 Swapping in new binary...
2020/11/25 14:59:32 Error upgrading: Unable to unzip downloaded archive: zip: not a valid zip file

```

4. Ran upgrade again but says it did complete the upgrade: 
```
notedo@notedoserver:/opt/notedo$ ./notedo upgrade
2020/11/25 15:02:46 Already up to date: 0.1.8-7-gdba1ef9
```

**Expected behavior**

**Additional context**
Add any other context about the problem here (screenshots, files, etc.)

![image](https://user-images.githubusercontent.com/1312111/100244641-c8c63000-2efc-11eb-94be-4c14cb082eb3.png)

Running ``` tar xvzf notedo-linux-0.1.8-7-gdba1ef9.tar.gz``` worked fine.

**Comment by davidwashere** [comment]
yeup, that is indeed a bug, will be fixed in release later today. Thanks!

**Comment by davidwashere** [comment]
This is fixed in `0.1.9`+, will need to do the upgrade manually (ie: untar the .tar.gz and replace the executable)

**Comment by davidwashere** [comment]
Fixed
